### DISCUSSION TITLE
r19base:/21.transition

### DISCUSSION THREAD

**Discussion by giscus** [yangbo5207/react19feature#33] [General]
# tutorial/use/transition

https://tailwindcss.online/tutorial/use/transition

**Comment by dlgchg** [comment]
我来了

**Comment by xiaoqufengdi** [comment]
降低 UI 任务更新的优先级，可否结合时间切片去理解，任务能排上的话，下个时间切片执行？

**Reply by yangbo5207** [reply]
可以。时间切片，任务优先级都是一套逻辑里的概念
